src/styles/navigation.scss

```diff
@@ -0,0 +1,55 @@
+.navigation {
+  margin-right: 50px;
+  font-family: "Roboto", sans-serif;
+
+  &__items-list {
+    display: flex;
+    align-items: center;
+    height: 60px;
+    margin: 0;
```

Comment:
It's good practice to reset or normalize default css. For example, use this [meyer css reset code](https://meyerweb.com/eric/tools/css/reset/)